perf_benchmark/benchmark_common.cc

```diff
@@ -0,0 +1,294 @@
+// Copyright 2016 Google Inc. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+////////////////////////////////////////////////////////////////////////////////
+//
+#include <fstream>
+#include <sstream>
+#include <utility>
+#include "perf_benchmark/benchmark_common.h"
+#include "google/protobuf/text_format.h"
+#include "absl/strings/escaping.h"
+#include "nlohmann/json.hpp"
+
+namespace google {
+namespace grpc {
+namespace transcoding {
+
+namespace perf_benchmark {
+
+namespace pb = ::google::protobuf;
+
+std::string LoadFile(const std::string& file_name) {
```

Comment:
pass around `absl::string_view` instead of `const std::string&` (?)

[go/totw/1](https://goto.google.com/totw/1)